contracts/apm/Repo.sol

```diff
@@ -27,6 +28,7 @@ contract Repo is AragonApp {
     */
     function initialize() public onlyInit {
         initialized();
+        versionsNextIndex = 1;
```

Comment:
Wonder if we could just use `versionsLength`, since we don't need to reserve the 0 slot?

Reply:
It is needed for [these reverse mappings](https://github.com/aragon/aragonOS/pull/430/files#diff-460eb60e27bc67ff984d38f5b91246fdR66) to work, so if you ask for a version with an inexistent semantic version or inexistent contract address, it reverts.

Reply:
:+1: